src/main/java/christmas/domain/Date.java

```diff
@@ -0,0 +1,58 @@
+package christmas.domain;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+
+public class Date {
+    public static final String DATE_RE_READ_REQUEST_MESSAGE = "유효하지 않은 날짜입니다. 다시 입력해 주세요.";
+    private static final LocalDate CHRISTMAS_DAY = LocalDate.of(2023, 12, 25);
+    private static final int YEAR = 2023;
+    private static final int MONTH = 12;
+    private static final int DATE_MIN_NUMBER = 1;
+    private static final int DATE_MAX_NUMBER = 31;
+
+    private final int dayNumber;
+
+    public Date(int dayNumber) {
+        validateRange(dayNumber);
+        this.dayNumber = dayNumber;
+    }
+
+    private void validateRange(int dayNumber) {
+        if ((dayNumber < DATE_MIN_NUMBER) || (dayNumber > DATE_MAX_NUMBER)) {
+            throw new IllegalArgumentException(DATE_RE_READ_REQUEST_MESSAGE);
+        }
```

Comment:
(이런 방법도 있어요!)

```suggestion
        try {
            LocalDate.of(YEAR, MONTH, dayNumber);
        } catch (NumberFormatException exception) {
            throw new IllegalArgumentException(DATE_RE_READ_REQUEST_MESSAGE);
        }
```

이런식으로 LocalDate를 활용하는 방법도 있습니다!

Reply:
오,, 이번에 LocalDate를 처음 사용해봐서 익숙하지 않았는데요 이런 방법도 있었군요,, 감사합니다!

Reply:
@JaeHongDev 
저도 새롭게 배워갑니다..!